### PR TITLE
RavenDB-19902 Add empty bucket list message

### PR DESCRIPTION
**Pull request by mateuszbartosik**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-19902/Buckets-report-for-empty-database

### Additional description

Added a message that says the bucket list is empty.

_Please delete below the options that are not relevant_

### Type of change
- New feature

### How risky is the change?
- Not relevant

### Backward compatibility
- Not relevant

### Is it platform specific issue?
- No

### Documentation update
- No documentation update is needed 

### Testing by Contributor
- It has been verified by manual testing

### Testing by RavenDB QA team
- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?
- No

### UI work
- No UI work is needed

### CODE REVIEW

File: src/Raven.Studio/wwwroot/App/views/database/status/bucketsReport.html [hunk 1]
```diff
@@ -1,72 +1,73 @@
 <div id="storage-report" class="content-margin">
-    <div id="storage-report-container" data-bind="css: { 'processing': showLoader() }">
-        <div class="chart-tooltip tooltip tooltip-inner" style="opacity: 0">
+    <!-- ko if: node().internalChildren.length != 0 -->
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/database/status/bucketsReport.html]: !==

File: src/Raven.Studio/wwwroot/App/views/database/status/bucketsReport.html [hunk 2]
```diff
@@ -1,72 +1,73 @@
...
+            </tbody>
+        </table>
+        <div data-bind="visible: showBucketContents">
+            <h3>Items stored in this bucket:</h3>
+            <div class="margin-bottom" style="position: relative; height: 50vw">
+                <virtual-grid style="height: 50vw" class="resizable flex-window" params="controller: gridController"></virtual-grid>
+            </div>
+        </div>
+    <!-- /ko -->
+    <!-- ko if: node().internalChildren.length == 0 -->
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/database/status/bucketsReport.html]: ===

**Review by ml054 — CHANGES_REQUESTED** [review]
Please have a look at my comments.

File: src/Raven.Studio/wwwroot/App/views/database/status/bucketsReport.html (hunk 1 above)
Comment (mateuszbartosik) [reply src/Raven.Studio/wwwroot/App/views/database/status/bucketsReport.html]: Fixed

File: src/Raven.Studio/wwwroot/App/views/database/status/bucketsReport.html (hunk 2 above)
Comment (mateuszbartosik) [reply src/Raven.Studio/wwwroot/App/views/database/status/bucketsReport.html]: Fixed

**Review by ml054 — APPROVED** [review]

**Pull request merged**